website/components/Theme.tsx

```
@@ -40,17 +40,23 @@ function Theme({ fonts = [], variables = [], colorScheme }: Props) {
 
   return (
     <Head>
-      {fonts.map(({ styleSheet }) => (
+      {fonts?.map(({ styleSheet }) => (
+        styleSheet
+          ? (
+            <style
+              type="text/css"
+              dangerouslySetInnerHTML={{ __html: styleSheet }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)